qiskit_dynamics/models/operator_collections.py

```diff
@@ -879,28 +984,30 @@ class JAXSparseLindbladCollection(BaseLindbladOperatorCollection):
     """
 
     @property
-    def static_hamiltonian(self) -> "BCOO":
+    def static_hamiltonian(self) -> BCOO:
```

Comment:
Ah I guess this answers the question as to why you need to define `BCOO` even if JAX can't be imported.